app/main.py

```diff
@@ -1,34 +1,77 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
+
+    def str(self) -> None:
```

Comment:
The method `str` should be renamed to `__str__` to properly override the string representation method in Python. This allows the `Deck` object to be converted to a string using `str(deck)` or `print(deck)`.